src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/Contracts/ExtendedDatabaseInfo.cs

```diff
@@ -0,0 +1,24 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+using System.Collections;
+using System.Collections.Generic;
+using Microsoft.SqlTools.ServiceLayer.Admin.Contracts;
+
+namespace Microsoft.SqlTools.ServiceLayer.DisasterRecovery.Contracts
+{
+    /// <summary>
+    /// Provides database info for backup/recovery. Extends the default DatabaseInfo
+    /// </summary>
+    public class ExtendedDatabaseInfo: DatabaseInfo
```

Comment:
>ExtendedDatabaseInfo [](start = 17, length = 20)

I don't think this is the best name for this class.  I think this should be a "Backup Configuration Info" class which contains a DatabaseInfo object..not really a derived class from DatabaseInfo.